pages/common/fswatch.md

```diff
@@ -4,12 +4,12 @@
 
 - Run a bash command on file creation, update or deletion:
 
-`fswatch -0 {{path_to_watch}} | xargs -0 {{bash_command}}`
+`fswatch -0 {{path/to/watch}} | xargs -0 {{bash_command}}`
 
 - Use `{}` in your bash command as a placeholder for the absolute path to the file change:
 
-`fswatch -0 {{path_to_watch}} | xargs -0 -I {} {{bash_command}}`
+`fswatch -0 {{path/to/watch}} | xargs -0 -I {} {{bash_command}}`
 
 - Filter by certain event type, eg. Updated, Deleted or Created:
 
-fswatch -0 --event {{event_type}} {{path_to_watch}} | xargs -0 {{bash_command}}`
+`fswatch -0 --event Updated {{path/to/watch}} | xargs -0 {{bash_command}}`
```

Comment:
You don't need to remove the `{{}}`. `{{Updated}}` is fine. Refer to - https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#token-syntax